tracking/index.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+<head>
+    <meta charset="utf-8">
+    <title>ViSearch Widget</title>
+    <meta name="description" content="A page to test the different widgets">
+    <meta name="viewport" content="width=device-width">
+    <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
+<!--    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">-->
+    <style type="text/css" media="screen, print">
+        /** Regular */
+        body { font-family: -apple-system, BlinkMacSystemFont, sans-serif; }
+    </style>
+    <!-- build:js scripts/vendor/modernizr.js -->
+    <!-- endbuild -->
+
+</head>
+<body onload="myFunction()">
+<!--[if lt IE 10]>
+<p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+<![endif]-->
+
+<div class="container">
+    <div class="header">
+        <ul class="nav nav-pills pull-right">
+            <li class="active"><a href="#">Home</a></li>
+        </ul>
+        <h3 class="text-muted">Visenze Widget</h3>
+    </div>
+
+    <div class="vs-image-body">
+        <img src="https://i.pinimg.com/originals/18/c1/fa/18c1fad69bd35a9f3bbacae3ad1dfcc2.jpg"> 
+    </div>
+
+</div>
+
+<script>
+    function myFunction() {
+        let tracker = va.init({code: "test"}); 
+
+        tracker.sendEvent("testClick", {
```

Comment:
this is not good demo, can we simulate a real scenario i.e. we have a div of the search results, then show the user what happen upon product click, impression. Also have a fake buy button to demonstrate Transaction event.

Reply:
changed test html page.

Reply:
can demo all events as commented?

Reply:
added 6 button to demo 6 defined events .